### DISCUSSION TITLE
OnlyM

### DISCUSSION THREAD

**Discussion by geostanko** [AntonyCorbett/OnlyM#441] [Q&A]
is OnlyM working? I tried the version before Covid-19 the other day so it didn't work for me ... should I download the new version?

**Comment by AntonyCorbett** [comment]
@geostanko There are no reported problems. When you say it "didn't work" please try to be more specific.

**Comment by geostanko** [comment]
Issue #442

    On Monday, April 11, 2022, 10:39:09 AM GMT+2, Antony Corbett ***@***.***> wrote:  
 
 


@geostanko There are no reported problems. When you say it "didn't work" please try to be more specific.

**Reply by mvpapen** [reply]
Hi Anthony.
Thank you for all the work you do in behalf of the Congregations. 
?. on my home pc ,latest win 11 version only M displays nearly all formats like, JPG.JPEG, pdf, mp4,mp3 and so on.
In our kingdom hall,(latest win 10) I created 4 user accounts and installed Only in all accounts.
But so far we can only drag and drop JPEG files and mp4. All other formats can not be dropped in. 
What am I doing wrong? For now we change all other formats with " Snip" to JPEG. I hope we can avoid this.

**Reply by AntonyCorbett** [reply]
@mvpapen - thanks for your report. Please would you give us more information regarding what happens? When you state "cannot be dropped in", do you mean that when you drag the file over the window, the cursor changes to the "no entry" symbol? Or is the file accepted by the OnlyM window but then cannot be read? Or does something else happen?

**Reply by mvpapen** [reply]
Update: I just tried only M on my 2np Pc with latest Win 10 edition and all seems to work fine.
At the hall when dragging it in It says : non supported file:  and no entry symbol and it will not let me drop it in other then a Jpeg.file
Can I delete and reinstall the app. can it be that only admin can drag and drop?

**Reply by AntonyCorbett** [reply]
@mvpapen I can't reproduce what you are seeing here. When a "no entry" cursor symbol is displayed, no error message is shown; it simply prevents you from dropping the file. I wonder if you may be attempting to drag and drop a shortcut? It may look like an MP4 file, but perhaps it's a shortcut to the file?

**Reply by mvpapen** [reply]
good point , we will test it

**Reply by mvpapen** [reply]
Hi Anthony. Sorry about the delay. You where right!. The Files which did not take where shortcuts and not proper files. We are instructing the friends to be aware when copying and sending files or Pics.
Thank you for your help.

**Reply by mvpapen** [reply]
Hi Br Anthony. Just this week we have an issue with the year text not coming back on after playing video or picture. Compatibility mode is on and always on top is of. multiple congregation are reaching out to me  they all noticed same issue also on my private pc i have same issue.
It happened after  the last Jwl update earlier this week . Did you notice that too? What can we do?

**Reply by mvpapen** [reply]
I was using only M

**Reply by AntonyCorbett** [reply]
The JWL update to v15 broke the compatibility mode in OnlyM. Can I suggest you use JwlMediaWin to fix this issue? I will update OnlyM asap, but the JwlMediaWin tool is a better solution

**Comment by mvpapen** [comment]
We are using JwIMediaWin the last 3 years.  I have it on auto startup and
it is in the task bar. Is there a newer version? Should  I re install?

**Comment by geostanko** [comment]
what is JwlMedia Win ? 
    On Sunday, December 15, 2024 at 11:02:30 PM GMT+1, Antony Corbett ***@***.***> wrote:  
 
 


The JWL update to v15 broke the compatibility mode in OnlyM. Can I suggest you use JwlMediaWin to fix this issue? I will update OnlyM asap, but the JwlMediaWin tool is a better solution

**Comment by geostanko** [comment]
found...ok
    On Sunday, December 15, 2024 at 11:02:30 PM GMT+1, Antony Corbett ***@***.***> wrote:  
 
 


The JWL update to v15 broke the compatibility mode in OnlyM. Can I suggest you use JwlMediaWin to fix this issue? I will update OnlyM asap, but the JwlMediaWin tool is a better solution
